mls-rs-uniffi/src/lib.rs

```diff
@@ -408,12 +439,12 @@ impl Client {
     ///
     /// See [`mls_rs::Client::create_group`] and
     /// [`mls_rs::Client::create_group_with_id`] for details.
-    pub async fn create_group(&self, group_id: Option<Vec<u8>>) -> Result<Group, Error> {
+    pub async fn create_group(&self, group_id: Option<GroupId>) -> Result<Group, Error> {
         let extensions = mls_rs::ExtensionList::new();
         let inner = match group_id {
             Some(group_id) => {
                 self.inner
-                    .create_group_with_id(group_id, extensions)
+                    .create_group_with_id(group_id.into(), extensions)
                     .await?
```

Comment:
We have group id bytes in a bunch of places in the main crate -- would it make sense to make it a type there as well, at least in the public API? A quick search gives `create_group_with_id(group_id, ...)`, `GroupState::id`, `GroupContext::group_id(&self)`, `CommitBuilder::reinit(group_id, ...)`

Reply:
Yes, I think it would — I'm happy to push the changes down to that level as well. The type can be a `struct GroupId(Vec<u8>)` there — UniFFI doesn't support record-structs without field names.

Reply:
Would you want me to push the change further down in the API stack in this PR?